OPT_mission.Altis/cba_settings_war.sqf

```diff
@@ -134,7 +126,7 @@ force DEFAULT_FIELDREPAIR_MAX_REP_TIME = 240;
 force DEFAULT_FREE_REFUELS = 1;
 force DEFAULT_FREE_REFUELS_DURATION = 60;
 force DEFAULT_FREE_REPAIRS = 1;
-force DEFAULT_FULLREPAIR_LENGTH = 120;
+force DEFAULT_FULLREPAIR_LENGTH = 60;
```

Comment:
why did this changed?

Reply:
Weil wir uns damals auf 60sec geeinigt hatten und es in der Trainingsversion schon so eingestellt ist. Habe es nur in der War-Version auch so angepasst.